pipelines/pca/Snakefile

```diff
@@ -0,0 +1,24 @@
+configfile: "config.yml"
+
+
+rule all:
+    input:
+        expand(
+            "../../data/pca/{species_group}.pca.zarr.zip",
```

Comment:
```suggestion
            "../../data/popstructure/ag3.{species_group}.individual_allele_counts.zarr.zip",
```

Naming to make clear this is input data to PCA, not PCA itself.

Reply:
noted- thanks